src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutor.cs

```diff
@@ -633,7 +682,35 @@ private static ITaskSeriesTimer StartParameterLogTimer(IRecurrentCommand updateC
                 await TryHandleTimeoutAsync(invokeTask, CancellationToken.None, throwOnTimeout, timeoutTokenSource.Token, timerInterval, instance, null);
             }
 
-            await invokeTask;
+            FunctionResult functionResult = null;
+
+            try
+            {
+                await invokeTask;
```

Comment:
As mentioned above, you'll want to include invocation of executing filters within this try/catch so your below error handling gets any errors.